src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/NameQuotedValuePairListEncoding.cs

```diff
@@ -0,0 +1,124 @@
+
+namespace Microsoft.VisualStudio.ProjectSystem.Debug
+{
+
+    [Export(typeof(INameValuePairListEncoding))]
+    [ExportMetadata("Encoding", "NameQuotedValuePairListEncoding")]
+    internal sealed class NameQuotedValuePairListEncoding : INameValuePairListEncoding
+    {
+        public IEnumerable<(string Name, string Value)> Parse(string value)
+        {
+            if (string.IsNullOrWhiteSpace(value))
+            {
+                yield break;
+            }
+
+            foreach (var entry in ReadEntries(value))
+            {
+                var (entryKey, entryValue) = SplitEntry(entry);
+
+                if (!string.IsNullOrEmpty(entryKey))
+                {
+                    yield return (DecodeCharacters(entryKey), StripQuotes(DecodeCharacters(entryValue)));
+                }
+            }
+
+            static IEnumerable<string> ReadEntries(string rawText)
+            {
+                bool escaped = false;
+                int entryStart = 0;
+                for (int i = 0; i < rawText.Length; i++)
+                {
+                    if (rawText[i] == ',' && !escaped)
+                    {
+                        yield return rawText.Substring(entryStart, i - entryStart);
+                        entryStart = i + 1;
+                        escaped = false;
+                    }
+                    else if (rawText[i] == '/')
+                    {
+                        escaped = !escaped;
+                    }
+                    else
+                    {
+                        escaped = false;
+                    }
+                }
+
+                yield return rawText.Substring(entryStart);
+            }
+
+            static (string EncodedKey, string EncodedValue) SplitEntry(string entry)
+            {
+                bool escaped = false;
+                for (int i = 0; i < entry.Length; i++)
+                {
+                    if (entry[i] == '=' && !escaped)
+                    {
+                        var name = entry.Substring(0, i);
+                        var value = entry.Substring(i + 1);
+                        if (name.Length == 0 || value.Length == 0)
+                        {
+                            throw new FormatException($"Expected valid name value pair.");
+                        }
+                        return (name , value);
+                    }
+                    else if (entry[i] == '/')
+                    {
+                        escaped = !escaped;
+                    }
+                    else
+                    {
+                        escaped = false;
+                    }
+                }
+                throw new FormatException("Expected valid name value pair.");
+            }
+
+            static string DecodeCharacters(string value)
+            {
+                
+                return value.Replace("/\"", "\"").Replace("/=", "=").Replace("/,", ",").Replace("//", "/");
```

Comment:
What happens if a character other than `"=,/` is escaped?

For example, given input `a/bc`, should it:

1. Return `abc` 
2. Return `a/bc`
3. Throw

Whatever you decide, I'd add a test case to cover it.